content/react/concepts/react-native/react-native-components/react-native-components.md

```diff
@@ -0,0 +1,159 @@
+---
+Title: 'React Native Components'
+Description: 'Bundles of reusable, nestable code used to describe the appearance and behavior of a UI.'
+Subjects:
+  - 'Mobile Development'
+Tags:
+  - 'Components'
+  - 'React Native'
+  - 'UI'
+CatalogContent:
+  - 'learn-react-native'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**React Native Components** are bundles of reusable, nestable code used to describe the appearance and behavior of a User Interface (UI).
+
+## UI Views
+
+The basic building block of a **UI** is the **View,** a small, rectangular and often times nestable element that can be used to display text, images and respond to user input. React Native works by invoking these Views in their native environment with JavaScript, using React Components.
+
+![React Native to Native Ui Representation](../../../../../media/react_native_UI_Views.png)
+
+## Native Components
+
+These previously mentioned platform-backed components are called **Native Components,** and React Native creates the corresponding, platform specific Views (whether iOS or Android) for these components at runtime. Because of this, React Native apps look, feel and perform like Native apps.
```

Comment:
```suggestion
These previously mentioned platform-backed components are called Native Components, and React Native creates the corresponding, platform specific Views (whether iOS or Android) for these components at runtime. Because of this, React Native apps look, feel and perform like Native apps.
```